### PR TITLE
[DOCS] Bug 1212 — change "The {kib} console" to "Console" throughout docs. 

### PR DESCRIPTION
**Pull request by bironsidecmd**
Addresses https://github.com/elastic/security-docs/issues/1212 by changing "The {kib} Console" to "Console". 

[Preview](https://security-docs_1234.docs-preview.app.elstc.co/guide/en/security/master/security-apis.html)

### CODE REVIEW

**Comment by benironside** [comment]
FYI I reached out to the CLA team to try to get the CLA check resolved. I've signed the CLA, but I made my first commit from the wrong Github account, for which I had not yet signed the CLA at the time I submitted the PR.

**Comment by jmikell821** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/cases/api/actions-api/cases-actions-api-intro.asciidoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ You can push {es-sec} cases to these third-party systems:
 * {jira} (including Jira Service Desk)
 * {ibm-r}
 
-NOTE: The {kib} Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (joepeeples) [inline docs/cases/api/actions-api/cases-actions-api-intro.asciidoc:11]: The target page has a new title now, and the URL can be revised to use the `{kibana-ref}` attribute, as we all discussed in team chat. You'll also need to update all the other files with this link, but I'll just mark up this instance and leave the rest to you.
```suggestion
NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
```

**Review by joepeeples — COMMENTED** [review]
Looks good, see comment about link text and URL. Thanks!

**Comment by jmikell821** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by nastasha-solomon — APPROVED** [review]
LGTM!

**Review by joepeeples — APPROVED** [review]

**Review by benironside — APPROVED** [review]

**Comment by jmikell821** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**